addons/website/static/src/js/editor/wysiwyg.js

```diff
@@ -29,7 +29,14 @@ function toggleDropdown($toggles, show) {
                 toShow ? 'shown.bs.dropdown' : 'hidden.bs.dropdown',
                 () => resolve()
             );
-            $toggle.dropdown(toShow ? 'show' : 'hide');
+            // We need the iframe dropdown instance to trigger _onDropdownShow.
+            const bsInstance =
+                $toggle[0].ownerDocument.defaultView.Dropdown.getOrCreateInstance($toggle[0]);
+            if (toShow) {
+                bsInstance.show();
+            } else {
+                bsInstance.hide();
+            }
```

Comment:
When testing this fix with the "Sub Menus" navbar option set to "On Hover", the alignment does not not done correctly.

![Capture d’écran de 2023-03-15 10-31-05](https://user-images.githubusercontent.com/52911687/225267504-a70d6b67-e8c4-4e7e-aaf3-c321002c2226.png)

Reply:
Good catch! As this bug is present since 14.0, I will sove it in [this other PR](https://github.com/odoo/odoo/pull/115153)